src/main/java/fr/adrienbrault/idea/symfony2plugin/dic/inspection/CaseSensitivityServiceInspection.java

```diff
@@ -91,6 +92,11 @@ public void visitElement(PsiElement psiElement) {
 
     private void phpVisitor(final @NotNull ProblemsHolder holder, @NotNull PsiFile psiFile) {
 
+        /* skip inspections since it doesn't make sense in SF 4, where service IDs are often FQNs */
```

Comment:
the actual reason for skipping on Symfony 4 is because ids are case sensitive in 4.x.

and this also applies to parameters, not only to services.